tests/acceptance/28_inform_testing/01_files/edit_line.cf.expected

```diff
@@ -0,0 +1,19 @@
+    info: Deleted the promised line 1 'foo 1' from /tmp/TEST.cfengine
+    info: delete_lines promise 'foo 1' repaired
+    info: Edited file '/tmp/TEST.cfengine'
+    info: Edit file '/tmp/TEST.cfengine'
+    info: Edit file '/tmp/TEST.cfengine'
+    info: Edit file '/tmp/TEST.cfengine'
+    info: Edit file '/tmp/TEST.cfengine'
```

Comment:
This is definitely a weird log message. Are you creating tickets for these things, @craigcomstock? I believe the goal here is to add tests and make sure things like this are fixed.

Reply:
Ah, I am sorting the results to get rid of "ordering", maybe I should revert that change. As for making tickets, no, I haven't considered too hard whether the output is correct or not. I will change back away from sorting so we can see the order, review and create appropriate tickets. :+1:

Reply:
I am wondering if it wouldn't be easier to have ONE promise per test/expected output. That way it would be clearer where one promise starts and another stops in the logs that are generated.